src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -87,7 +86,6 @@ public DefaultCaseDefinitionRepository(final ApplicationParams applicationParams
     }
 
     @Override
-    @Cacheable("caseTypeDefinitionsCache")
     public CaseTypeDefinition getCaseType(int version, String caseTypeId) {
```

Comment:
what's the use of version here? if we remove cache annotation?

Reply:
We actually use the version in the cached class, this is just a overridden method